ci/pod/docker-compose.plugin.yml

```diff
@@ -324,10 +324,21 @@ services:
     networks:
       vector_net:
 
+  clickhouse:
+    image: clickhouse/clickhouse-server:23.4.2-alpine
+    container_name: clickhouse
+    ports:
+      - '8123:8123'
+      - '9000:9000'
```

Comment:
why create 9000? I only find the port 8123 port has been used.

Reply:
Just in case we need those ports in the future, we will not have to redeclare it again. That's why I bound all the ports available.

Reply:
one PR only for one thing I think.